app/views/static_pages/index.html.erb

```diff
@@ -1,34 +1,23 @@
-
-
-<h1>Welcome to Intergalactic Battleship!</h1>
-
-<p>may the binks be with you.</p>
-
-<!--insert create game button here -->
-
-<div>
-<%= link_to 'New Game', new_game_path, class: 'btn btn-default'%>
-
-<%= link_to 'Join A Game', games_path, class: 'btn btn-default'%>
-</div>
+<!-- Static Index Page -->
 
 <div id="landing_page_image">
 	<img src="/assets/background_landing.png">
 </div>
 <div class="col-xs-10 col-xs-offset-1 features text-center">
 	<div class="lpage_header">
-		<h1>STAR WARS: BATTLESHIP</h1>
+		<h1>Intergalactic Battleship</h1>
 	</div>
 </div>
 <div class="col-xs-10 col-xs-offset-1 features text-center">
-	<div class="col-sm-4">
-		<a href="#"><h2>New Game <br />(Player 1)</h2></a>
-	</div>
-	<div class="col-sm-4">
-		<a href="#"><h2>Join a Game <br />(Player 2)</h2></a>
+	<div class="col-sm-6">
+		<%= link_to new_game_path do %>
+			<h2>New Game<br />(Player 1)</h2>
```

Comment:
Minor nitpick, but the `br` tag within the `h2` isn't ideal.  I'd probably suggest to have 2 different `h2`s